packages/amplify_auth_cognito/example/lib/main.dart

```diff
@@ -346,6 +346,24 @@ class _MyAppState extends State<MyApp> {
     }
   }
 
+  void _getCurrentUser() async {
+    setState(() {
+      error = "";
+      exceptions = [];
+    });
+    try {
+      AuthUser res = await Amplify.Auth.getCurrrentUser();
+      print(res);
+    } on AuthError catch (e) {
+      setState(() {
+        error = e.cause;
+        e.exceptionList.forEach((el) {
```

Comment:
`el` is too short -- but the real problem is that it looks like __E__exception __L__ist, which is a collection of many things. Since this is a lambda argument, iterating over the list, this variable should be _singular_ in some way. Perhaps `forEach((exception) {`, etc.

Reply:
Same as previous comment.